demisto_sdk/tests/validate_content_path_test.py

```diff
@@ -33,6 +35,33 @@
 )
 
 
+@pytest.mark.parametrize(
+    "suffix",
+    ("json", "yml"),
+)
+def test_xdrc_template_file_valid(suffix: str):
+    folder = "foo"
+    _validate(DUMMY_PACK_PATH / XDRC_TEMPLATE_DIR / folder / f"{folder}.{suffix}")
+
+
+@pytest.mark.parametrize(
+    "file, suffix",
+    (
+        ("MyXDRCTemplate_test", "json"),
+        ("MyXDRCTemplate_test", "yml"),
+        ("MyXDRCTemplate", "py"),
+    ),
+)
+def test_xdrc_template_file_invalid(file: str, suffix: str):
+    """
+    The first two tests will fail because the file name is incorrect
+    The last one will fail because the file extension is incorrect (should be .json or .yml)
```

Comment:
please add this information to the test cases using `pytest.param` and `id='bad name, good suffix'` etc